File: Level1/numberOfpy.cpp
#include <string>
#include <iostream>
using namespace std;

bool solution(string s)
{
    bool answer = true;
    int p_count = 0;
    int y_count = 0;
    for (int i = 0; i < s.size(); i++) {
        s[i]=tolower(s[i]);
    }
    for (int i = 0; i < s.size(); i++) {
        if (s[i] == 'p') p_count++;
        if (s[i] == 'y') y_count++;
    }
    if (p_count != y_count) answer = false;
    
    return answer;
}

File: Programmers/Level1/Level1_practice/numberOfpy.py
def solution(s):
    answer = True
    p=0
    y=0
    for i in range(len(s)):
        if s[i]=='P' or s[i]=='p': p+=1
        elif s[i]=='Y' or s[i]=='y': y+=1
    if(p==y): answer=True
    else: answer=False
    return answer

if __name__ =='__main__':
    print(solution('Pyy'))